### PR TITLE
Sphinx documentation. Fixes #8

### LINKED ISSUE #8: Docs to RTD

### PR DESCRIPTION
**Pull request by jamadden**
Can be viewed at http://zopesendmail.readthedocs.io/en/docs/

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -7,5 +7,8 @@ include .travis.yml
 include .coveragerc
 
 recursive-include src *
+recursive-include docs *py
+recursive-include docs *rst
```
Comment (mgedmin) [inline MANIFEST.in]: (These look weird without the dots.)

Comment (jamadden) [reply MANIFEST.in]: OK, changed :)

**Comment by jamadden** [comment]
Thank you! Could we please get a PyPI release when convenient (or access for same)?

**Comment by mgedmin** [comment]
Whoops, should've checked myself!

You now have PyPI rights to zope.sendmail.

**Comment by jamadden** [comment]
Thank you! 4.1.0 has been released.

**Pull request merged**